spacy/tokens/doc.pyx

```diff
@@ -1743,6 +1743,124 @@ cdef class Doc:
                     j += 1
         return output
 
+    @cython.boundscheck(False)  # Deactivate bounds checking
+    def get_character_combination_hashes(self,
+        *,
+        const bint cs, 
```

Comment:
Rename to `case_sensitive`?

Reply:
The aim here was to prevent commands from having to be distributed over too many lines and so becoming hard to read. Which do you think is more important: the readability of the variables or the readability of the commands? I don't really have a strong opinion either way.

Reply:
I had to read the code to understand what `cs` is, so I think it's better to be explicit.